sycl/CMakeLists.txt

```diff
@@ -218,6 +218,7 @@ add_custom_target( sycl-toolchain
           llvm-link
           llvm-objcopy
           sycl-post-link
+          sycl-ls
```

Comment:
Update SYCL_TOOLCHAIN_DEPLOY_COMPONENTS as well if it is supporsed to be delivered with DPCPP toolchain.

Reply:
OK